controllers/ArticleController.js

```diff
@@ -343,6 +344,108 @@ class ArticleController {
     }
     return res.status(200).json({ status: 200, articles: { ...articles, pages } });
   }
+
+  /**
+   * @author Caleb
+   * @param {Object} req
+   * @param {Object} res
+   * @param {*} next
+   * @returns {Object} Returns the response
+   */
+  static async shareArticleTwitter(req, res) {
+    const { slug } = req.params;
+    const article = await Article.findOne({
+      where: {
+        slug,
+        status: { [Op.not]: 'deleted' }
+      }
+    });
+    if (!article || article.status === 'unpublished') {
+      return res.status(404).json({
+        status: 404,
+        message: 'Article not found'
+      });
+    }
+    opn(`https://twitter.com/intent/tweet?text=${process.env.FRONTEND_URL}/articles/${slug}`);
+    return res.status(200).json({ status: 200, message: 'Sharing article via witter' });
```

Comment:
I can see a typo error ```witter``` instead of ```Twitter```

Reply:
@Nziranziza It is official twitter URL for sharing

Reply:
@mcaleb808 Daniel is talking about the response message `Sharing article via witter`, it should be `Sharing article via Twitter`

Reply:
@oesukam let me work on that!